### ISSUE TITLE
Mention that all SVG files are under the SIL OFL license

### ISSUE THREAD

**Issue by janhohner** [ForkAwesome/Fork-Awesome#62]
Hi,
I prefer to use the SVG versions of the icons supplied in /src/icons/svg. I'm just not sure which license applies to them. The website footer states "Code is licensed under MIT License" but the README states that only "Fork Awesome CSS, LESS, and Sass files are licensed under the MIT License". Could you please clarify this? Thanks!

**Comment by xuv** [comment]
Hello

Sorry for the late reply and thank you for your message and question. It's actually a good question and I'm not 100% sure on the answer. 

As you maybe know, Fork Awesome is a fork of Font Awesome 4.7. And Font Awesome 4.7 is distributed with 3 licenses that we have kept intact:
```
Fork Awesome is licensed under SIL OFL 1.1 · Code is licensed under MIT License · Documentation is licensed under CC BY 3.0
```
The thing we did is extract the SVG icons from the font into individual files. So since the font was released under a [SIL OFL 1.1](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL_web) license, I'm guessing the resulting SVG icons must be kept under that license. (MIT is for the code only)

Now, point 5 of the SIL license says this:
```
5) The Font Software, modified or unmodified, in part or in whole,
must be distributed entirely under this license, and must not be
distributed under any other license. The requirement for fonts to
remain under this license does not apply to any document created
using the Font Software.
```

The question is: are the individual SVG icons considered _a part of_ the original in font, and if such, they should be kept under the SIL OFL 1.1 license. Or are they considered documents created using the font software and thus could be licensed in any way we like. I tend to lean on the former, keeping the SIL license. But if you prefer the latter, consider the designs then under the documentation license, CC BY 3.0.
